### PR TITLE
fix bugs from the simple froms migration

### PR DESCRIPTION
**Pull request by cloudmagic80**
## Summary
This PR is to fix a bug that was created during the migration from simple_forms folder to the ivc_champva folder. Some variables were not changed.

Related to:
https://github.com/department-of-veterans-affairs/vets-api/pull/16091/files

## Testing done

- [ ] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
- *If this work is behind a flipper:*
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## What areas of the site does it impact?
This only impacts the ivc_champava forms

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -958,6 +958,7 @@ modules/dhp_connected_devices @department-of-veterans-affairs/digital-health-pla
 modules/facilities_api @department-of-veterans-affairs/vfs-facilities-frontend @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 modules/health_quest @department-of-veterans-affairs/vsa-healthcare-health-quest-1-backend
 modules/income_limits @department-of-veterans-affairs/vfs-public-websites-frontend @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+modules/ivc_champva/app/services/ivc_champva/pdf_filler.rb @department-of-veterans-affairs/champva-engineering @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
```
Comment (balexandr) [inline .github/CODEOWNERS:961]: We shouldn't need this. I added the entire module in one line so we don't have to do individual lines for each file

Comment (cloudmagic80) [reply .github/CODEOWNERS]: Its still giving me an error message. I have to add it

**Review by reiting — APPROVED** [review]

**Review by balexandr — DISMISSED** [review]
Lgtm

**Comment by rmtolmach** [comment]
@cloudmagic80 not sure what happened in your latest commit, but there are zero files changed in this PR now 😕

**Comment by rmtolmach** [comment]
@cloudmagic80 I converted this to a draft. Mark it as ready for review once there's a diff to review.

**Comment by RachalCassity** [comment]
@cloudmagic80 Can this PR be closed?

**Pull request closed without merging**